recipe-dsl/src/test/java/com/ing/baker/recipe/javadsl/RecipeTest.java

```diff
@@ -65,9 +65,9 @@ public void shouldSetupRecipeWithMultipleSieveDescriptors() {
         assertEquals(recipe.getEvents().size(), 0);
         assertEquals(recipe.getInteractions().size(), 0);
         assertEquals(recipe.getSieves().size(), 3);
-        assertEquals(recipe.getSieves().get(0).interaction(), requiresProcessIdStringInteractionCheck());
-        assertEquals(recipe.getSieves().get(1).interaction(), firesTwoEventInteractionCheck());
-        assertEquals(recipe.getSieves().get(2).interaction(), providesIngredientInteractionCheck());
+//        assertEquals(recipe.getSieves().get(0), requiresProcessIdStringInteractionCheck());
+//        assertEquals(recipe.getSieves().get(1), firesTwoEventInteractionCheck());
+//        assertEquals(recipe.getSieves().get(2), providesIngredientInteractionCheck());
```

Comment:
What are we going to do with these?

Reply:
I will remove these since they are no longer applicable.